content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,79 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing ways to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+Note that to start using a GDrive remote, you only need to add it with a valid
+URL (see next section), and use any DVC command that requires it (see
+`dvc pull`, `dvc fetch`, `dvc push`). For example:
+
+```dvc
+$ dvc remote add -d myremote gdrive://root/dvc/dvcstore
+$ dvc add file.dat
+$ dvc push -r myremote
+...
+Go to the following link in your browser:
+
+    https://accounts.google.com/o/oauth2/auth # ... copy this link
+
+Enter verification code: # <- enter resulting token
+```
+
+See the [Authorization](#authorization) section for more details.
+
+## URL format for Google Drive remotes
+
+A GDrive remote URLs is needed for the `dvc remote add` command. It can be
+constructed with a _base_, and an **optional** _path_ to a folder i.e.
+`gdrive://<base>/path/to/folder`. The base can be one of:
+
+1. `root` - indicates your topmost Google Drive directory.
+
+```dvc
+$ dvc remote add -d mygdroot gdrive://root
+$ dvc remote add mygdfolder gdrive://root/path/to/folder
+```
+
+2. Google Drive _Folder ID_
+
+To obtain the unique _Folder ID_ in question, navigate into that folder in your
+web browser, and find it in the address bar. For example, for
+`https://drive.google.com/drive/u/0/folders/0AIac4JZqHhKmUk9PDA`:
+
+```dvc
+$ dvc remote add mygdfolder gdrive://0AIac4JZqHhKmUk9PDA
+$ dvc remote add mygdsubfolder \
+                 gdrive://0AIac4JZqHhKmUk9PDA/sub/path
+```
+
+Note that [shared folders](https://support.google.com/drive/answer/7166529) and
+[shared drives](https://support.google.com/a/users/answer/9310351) can only be
+referenced by _Folder ID_.
+
+3. `appDataFolder` -
+   [special hidden folder](https://developers.google.com/drive/api/v2/appdata)
+   (unique per user) meant to store application-specific data. This is a good
+   choice to prevent accidentally deleting remote storage data from the Google
+   Drive web UI.
+
+```dvc
+$ dvc remote add mygdappata gdrive://appDataFolder
+```
+
+## Configure custom Google Cloud project & app (recommended)
```

Comment:
we should mention it briefly at least in the intro? 

also, I still don't like the title.

Ideally it should go from the task people would probably trying to solve with this ... not sure how to phrase this in a simple way

How about:

Using custom project
Use custom credentials
Connect with custom project
etc

(recommended) - better to highlight it here, or in the note on the top that mentions this section and provides and why would I need this

Reply:
Alright. Title updated and note added in intro to this section, as well as to the service account section.

Reply:
p.s. the intro is kinda long now...

Reply:
yep, will check it again